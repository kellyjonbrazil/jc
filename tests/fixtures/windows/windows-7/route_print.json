{
  "interface_list": [
    {
      "interface_index": 10,
      "mac_address": "00:0c:29:86:1e:1f",
      "description": "Intel(R) PRO/1000 MT Network Connection"
    },
    {
      "interface_index": 1,
      "mac_address": null,
      "description": "Software Loopback Interface 1"
    },
    {
      "interface_index": 11,
      "mac_address": null,
      "description": "Microsoft ISATAP Adapter"
    }
  ],
  "ipv4_route_table": {
    "active_routes": [
      {
        "network_destination": "0.0.0.0",
        "netmask": "0.0.0.0",
        "gateway": "192.168.22.1",
        "interface": "192.168.22.33",
        "metric": "10"
      },
      {
        "network_destination": "127.0.0.0",
        "netmask": "255.0.0.0",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "306"
      },
      {
        "network_destination": "127.0.0.1",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "306"
      },
      {
        "network_destination": "127.255.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "306"
      },
      {
        "network_destination": "192.168.22.0",
        "netmask": "255.255.255.0",
        "gateway": "On-link",
        "interface": "192.168.22.33",
        "metric": "266"
      },
      {
        "network_destination": "192.168.22.33",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "192.168.22.33",
        "metric": "266"
      },
      {
        "network_destination": "192.168.22.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "192.168.22.33",
        "metric": "266"
      },
      {
        "network_destination": "224.0.0.0",
        "netmask": "240.0.0.0",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "306"
      },
      {
        "network_destination": "224.0.0.0",
        "netmask": "240.0.0.0",
        "gateway": "On-link",
        "interface": "192.168.22.33",
        "metric": "266"
      },
      {
        "network_destination": "255.255.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "306"
      },
      {
        "network_destination": "255.255.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "192.168.22.33",
        "metric": "266"
      }
    ],
    "persistent_routes": []
  },
  "ipv6_route_table": {
    "active_routes": [
      {
        "interface": 1,
        "metric": "306",
        "network_destination": "::1/128",
        "gateway": "On-link"
      },
      {
        "interface": 10,
        "metric": "266",
        "network_destination": "fe80::/64",
        "gateway": "On-link"
      },
      {
        "interface": 10,
        "metric": "266",
        "network_destination": "fe80::c447:d1ef:c29f:c48c/128",
        "gateway": "On-link"
      },
      {
        "interface": 1,
        "metric": "306",
        "network_destination": "ff00::/8",
        "gateway": "On-link"
      },
      {
        "interface": 10,
        "metric": "266",
        "network_destination": "ff00::/8",
        "gateway": "On-link"
      }
    ],
    "persistent_routes": []
  }
}
