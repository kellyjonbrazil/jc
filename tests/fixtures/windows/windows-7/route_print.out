===========================================================================
Interface List
 10...00 0c 29 86 1e 1f ......Intel(R) PRO/1000 MT Network Connection
  1...........................Software Loopback Interface 1
 11...00 00 00 00 00 00 00 e0 Microsoft ISATAP Adapter
===========================================================================

IPv4 Route Table
===========================================================================
Active Routes:
Network Destination        Netmask          Gateway       Interface  Metric
          0.0.0.0          0.0.0.0     192.168.22.1    192.168.22.33     10
        127.0.0.0        255.0.0.0         On-link         127.0.0.1    306
        127.0.0.1  255.255.255.255         On-link         127.0.0.1    306
  127.255.255.255  255.255.255.255         On-link         127.0.0.1    306
     192.168.22.0    255.255.255.0         On-link     192.168.22.33    266
    192.168.22.33  255.255.255.255         On-link     192.168.22.33    266
   192.168.22.255  255.255.255.255         On-link     192.168.22.33    266
        224.0.0.0        240.0.0.0         On-link         127.0.0.1    306
        224.0.0.0        240.0.0.0         On-link     192.168.22.33    266
  255.255.255.255  255.255.255.255         On-link         127.0.0.1    306
  255.255.255.255  255.255.255.255         On-link     192.168.22.33    266
===========================================================================
Persistent Routes:
  None

IPv6 Route Table
===========================================================================
Active Routes:
 If Metric Network Destination      Gateway
  1    306 ::1/128                  On-link
 10    266 fe80::/64                On-link
 10    266 fe80::c447:d1ef:c29f:c48c/128
                                    On-link
  1    306 ff00::/8                 On-link
 10    266 ff00::/8                 On-link
===========================================================================
Persistent Routes:
  None